web/client/components/I18N/FlagButton.jsx

```diff
@@ -27,7 +26,8 @@ class LangBar extends React.Component {
     static defaultProps = {
         locales: LocaleUtils.getSupportedLocales(),
         code: 'en-US',
-        onLanguageChange: function() {}
+        onLanguageChange: function() {},
+        onFlagSelected: () => {}
```

Comment:
What's the difference between the two events?

Reply:
I don't think there is difference,
onLanguageChange and onFlagSelected was both there before I changed it.
I only added onFlagSelected in default to prevent error if missing.
Should I remove onLanguageChange?
it doesn't seem used by the component.

Reply:
ok, got it, I would it leave it there